### ISSUE TITLE
Image component has two wrapper divs that can't be styled or removed

### ISSUE THREAD

**Issue by AleksandrHovhannisyan** [vercel/next.js#18820]
# Bug report

## Describe the bug

The `next/image` component renders unnecessary wrapper `<div>`s, with no API to style these or remove them from the DOM. This leads to:

1. Accessibility errors/HTML validation errors. Divs cannot be children of certain elements (e.g., buttons). Images can be.
2. Layout issues. It's not possible to position the images where you want them because of the divs.

## To Reproduce

1. Create a simple Next.js app with no dependencies.
2. Import and render the `next/image` component.
3. Inspect the DOM and note that there are wrapper divs around the `<img`> element.

## Expected behavior

Ideally, the wrapper divs should not be there. A `<picture>` element with nested `<img>` and `<source>`s would make more sense, with an exposed API to allow us to customize the picture parent and the nested image. If this is not possible, there should at least be props for the wrapper divs so we can position them accordingly.

## Screenshots

![image](https://user-images.githubusercontent.com/19352442/98179716-62398d80-1ecd-11eb-9af6-73cf72a814bc.png)

## System information

- OS: Windows
- Browser (if applies): Chrome
- Version of Next.js: 10.0.0

## Additional context

See discussion [here, on the Image component RFC](https://github.com/vercel/next.js/discussions/16832#discussioncomment-118288)

**Comment by Timer** [comment]
Have you tried Next.js 10.0.1? There's only one wrapping div now.
